### PR TITLE
No-ICU fallback.

### PR DESCRIPTION
**Pull request by lemire**
fixes https://github.com/ada-url/ada/issues/117

### CODE REVIEW

File: tests/wpt_tests.cpp [hunk 1]
```diff
@@ -479,7 +479,13 @@ int main(int argc, char** argv) {
   std::cout << "==============="<< std::endl;
   std::cout << "Final report: "<< std::endl;
   std::cout << "==============="<< std::endl;
-
+#if ADA_HAS_UCI
+  std::cout << "We are using UCI."<< std::endl;
```
Comment (anonrig) [inline tests/wpt_tests.cpp:483]: This should be ICU

File: tests/wpt_tests.cpp [hunk 2]
```diff
@@ -479,7 +479,13 @@ int main(int argc, char** argv) {
...
+#elif defined(_WIN32)
+  std::cout << "We are using Microsoft's Normaliz."<< std::endl;
+#else
+  std::cout << "UCI is unavailable and we have no feedback."<< std::endl;
```
Comment (anonrig) [inline tests/wpt_tests.cpp:487]: This should be ICU

File: tests/wpt_tests.cpp [hunk 3]
```diff
@@ -452,14 +452,14 @@ int main(int argc, char** argv) {
   name = "setters_tests_encoding("+std::string(SETTERS_TESTS_JSON)+")";
   if(all_tests || name.find(filter) != std::string::npos) {
     results[name] = setters_tests_encoding(SETTERS_TESTS_JSON);
-#ifdef _WIN32
+#if !ADA_HAS_UCI
```
Comment (anonrig) [inline tests/wpt_tests.cpp:455]: This needs to be Ada has icu

File: tests/wpt_tests.cpp (hunk 1 above)
Comment (lemire) [reply tests/wpt_tests.cpp:483]: Damn it.

Comment (lemire) [reply tests/wpt_tests.cpp]: Fixed.

**Review by anonrig — APPROVED** [review]

**Pull request merged**